packages/vite/src/node/publicUtils.ts

```diff
@@ -12,16 +12,12 @@ export {
   defaultAllowedOrigins,
 } from './constants'
 export { version as esbuildVersion } from 'esbuild'
-export {
-  splitVendorChunkPlugin,
-  splitVendorChunk,
-  isCSSRequest,
-} from './plugins/splitVendorChunk'
 export {
   normalizePath,
   mergeConfig,
   mergeAlias,
   createFilter,
+  isCSSRequest,
```

Comment:
We need to move `isCSSRequest` as we were duplicating it in `splitVendorChunk` to be able to export it in `publicUtils`. We could move in another PR other utils like `isCSSModule` to utils too.